Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
Binary files wxorig/src/.DS_Store and wxnew/src/.DS_Store differ
Binary files wxorig/src/osx/carbon/.DS_Store and wxnew/src/osx/carbon/.DS_Store differ
diff -uNr wxorig/src/osx/carbon/filedlg.cpp wxnew/src/osx/carbon/filedlg.cpp
--- wxorig/src/osx/carbon/filedlg.cpp	2010-11-05 08:58:50.000000000 -0400
+++ wxnew/src/osx/carbon/filedlg.cpp	2010-12-30 16:32:32.000000000 -0500
@@ -359,7 +359,7 @@
             while ( tokenizer.HasMoreTokens() )
             {
                 wxString extension = tokenizer.GetNextToken() ;
-                if ( extension.GetChar(0) == '*' )
+                if (extension.length() && extension.GetChar(0) == '*' )
                     extension = extension.Mid(1) ;
                 extension.MakeUpper();
 
